walker-backend/src/controllers/users.ts

```diff
@@ -9,9 +9,11 @@ class UserController {
     if (unique) {
       return res.status(400).json(`Account with email ${body.email} already exists.`)
     }
-    const userObj = body.user;
+    const user = new User({person_name: body.person_name,
```

Comment:
since we're changing the body here, it also needs to be changed [here](https://github.com/CMU-17-356/cmu-17-356-final-project-s23-walker/blob/master/walker/src/pages/Create.tsx#L30)